proposals/PickAPI/README.md

```diff
@@ -0,0 +1,192 @@
+        }
+    }
+}
+```
+
+### Room (Retargeting, Inheritance, and Invisibility)
+This example demonstrates custom picking behavior for several scopes.
+- Use retargeting to suppress picking of prims
+- Overriding retargeting when nesting `PickAPI` application
+- Using guide geometry as a target for unpickable prims
+
+```
+# The room specifies that by default, prims are visible to picking (if imaged)
+# but that they should retarget to an empty list. By default, objects in the
+# room act as pick occluders.
+def Xform "Room" (
+    append apiSchemas = "PickAPI"
+) {
+    uniform token pick:visibility = "inherited"
+    uniform token pick:retargeting = "replace"
+    rel pick:targets = []
+
+    def Xform "Walls" {
+        def Mesh "NorthWall" { ... }
+        def Mesh "SouthWall" { ... }
+        def Mesh "EastWall" { ... }
+        def Mesh "WestWall" { ... }
+    }
+
+    # The door overrides the retargeting behavior so that it and all its
+    # descendants are pickable and retarget to the rotatable 'DoorGroup'
+    def Xform "DoorGroup" (
+        append apiSchemas = "PickAPI"
+    ) {
+        uniform token[] xformOpOrder = ["xformOp:rotateZ"]
+        double xformOp:rotateZ = 0.0
+
+        uniform token pick:visibility = "inherited"
+        uniform token pick:retargeting = "replace"
+        rel pick:targets = <.>
+
+        def Mesh "Door" { ... }
+        def Mesh "DoorKnob" { ... }
+        def Mesh "DoorKnocker" { ... }
+    }
+
+    # The window makes its glass not pickable by default but provides a
+    # tiny sphere collocated with Glass to be used as a pick target by
+    # users when guides are enabled.
+    def Xform "Window" {
+        def Mesh "Frame" { ... }
+        def Mesh "Glass" (
+            append apiSchemas = "PickAPI"
+        ) {
+            uniform token pick:visibility = "invisible"
+        }
+        def Sphere "GlassPickTarget" (
+            append apiSchemas = "PickAPI"
+        ) {
+            uniform token purpose = "guide"
+            uniform token pick:visibility = "inherited"
+            uniform token pick:retargeting = "replace"
+            rel pick:targets = <../Glass>
+        }
+    }
+}
+```
+
+## Questions
+* This proposal does not address support for `UsdGeomSubset`. Visibility and purpose are currently not supported by subsets, so it may be hard to support the visibility attribute. However, one can imagine a desire for using face sets to describe pick retargeting.
+* Should pick visibility and and pick retargeting be two separate APIs? One could imagine tools providing facility for controlling pick visibility but not supporting retargeting. One could also imagine prims like `UsdGeomSubset` not supporting pick visibility but supporting pick retageting. One challenge with this formulation is identifying whether the visibility or retargeting schemas are responsible for describing pick occluders without creating ambiguity.
+* Is `UsdUI` the right place for viewport behavior schemas?
```

Comment:
It feels similar to `UsdUINodeGraphNodeAPI` to me, in that it's embedding extra information to guide a certain class of user interactions that have no bearing on "core computation or rendering"?